include/crow/query_string.h

```diff
@@ -51,7 +51,6 @@ char * qs_scanvalue(const char * key, const char * qs, char * val, size_t val_le
 
 inline int qs_strncmp(const char * s, const char * qs, size_t n)
 {
-    int i=0;
```

Comment:
again, this might be useful later, so let's just comment it

Reply:
see above, then we should leave it as it is.